pages/api/csv-export.js

```diff
@@ -1,4 +1,5 @@
 import fire from '../../config/firebaseAdminConfig';
+const ObjectsToCsv = require('objects-to-csv');
```

Comment:
Can this be imported using the `import` statement for consistency?

Reply:
tis done.